debian/package.json

```diff
@@ -10,18 +10,20 @@
     "player": "./bin/player-window",
     "production": "NODE_ENV=production electron .",
     "build": "NODE_ENV=production NEW_VERSION=$npm_package_version ./bin/build",
-    "rebuild": "electron-rebuild"
+    "install": "electron-rebuild"
```

Comment:
I didn't know there was a `postinstall` script (I'm still a newbie). I just tested it and it's working great. I'll push the commit in a minute